common/constants.ts

```diff
@@ -253,6 +253,23 @@ export const SEARCH_PIPELINE_DOCS_LINK =
 export const ML_RESPONSE_PROCESSOR_EXAMPLE_DOCS_LINK =
   'https://opensearch.org/docs/latest/search-plugins/search-pipelines/ml-inference-search-response/#example-externally-hosted-text-embedding-model';
 
+// Large Language Models Documentation Links
+export const BEDROCK_CLAUDE_3_SONNET_DOCS_LINK = 
+  'https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_converse_blueprint.md';
+
+export const OPENAI_GPT35_DOCS_LINK = 
+  'https://github.com/opensearch-project/ml-commons/blob/2.x/docs/remote_inference_blueprints/open_ai_connector_chat_blueprint.md';
+
+export const DEEPSEEK_CHAT_DOCS_LINK = 
+  'https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/deepseek_connector_chat_blueprint.md';
+
+// Embedding Models Documentation Links
+export const COHERE_EMBEDDING_MODEL_DOCS_LINK = 
+  'https://github.com/opensearch-project/ml-commons/blob/2.x/docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md';
+
+export const BEDROCK_TITAN_EMBEDDING_DOCS_LINK = 
+  'https://github.com/opensearch-project/ml-commons/blob/2.x/docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md';
+
```

Comment:
These are the blueprints we want to avoid customers visiting, as they are not compatible with ML inference processors, and generally do not have model interfaces, which means they are not compatible with OpenSearch Flow. We need to point to https://github.com/opensearch-project/dashboards-flow-framework/blob/main/documentation/models.md

For any models not present in that file ^, then suggest to add.